interpretation/ww_head.py

```diff
@@ -154,6 +154,22 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
     if isGameMaster == True:
         help_msg += "\n__Game Master commands:__\n"
 
+        '''forcesignup'''
```

Comment:
what does this have to do with the website?

Reply:
Not much. I still had this function written and merged it to the virtual machine already, but the common repository didn't have this yet.

...you could sync with the virtual machine first, if you want. This update should then disappear from this specific merge request.